posts/alpine-stack-size.md

```diff
@@ -0,0 +1,122 @@
+# An Alpine linux gotcha and what it means for you
+
+If you read Wikipedia you will find that Alpine is a GNU/Linux distribution that is based on musl (more on this later) and BusyBox.
```

Comment:
@naliwajek I did some more thinking - actually Alpine isn't a GNU/Linux distribution since GNU is the toolchain bundled in a distribution (which in Alpine's case isn't GNU libc - glibc)